nestedbox_type.admin.inc

```diff
@@ -78,3 +78,37 @@ function nestedbox_type_form_submit_delete(&$form, &$form_state) {
   $nestedbox_type = $form_state['nestedbox_type'];
   $form_state['redirect'] = 'admin/structure/nestedbox-types/manage/' . $nestedbox_type->type . '/delete';
 }
+
+/**
+ * Nestedbox type delete form.
+ */
+function nestedbox_type_form_delete_confirm($form, &$form_state, $task_type) {
+  if (!empty($task_type)) {
```

Comment:
I think the wildcard loader `nestedbox_type_load()` will make sure this is never empty.